docs/sdks/remove-api-key.md

```diff
@@ -31,11 +31,29 @@ from c8 import C8Client
 client = C8Client(protocol='https', host='gdn.paas.macrometa.io', port=443,
                         email='nemo@nautilus.com', password='xxxxx',
                         geofabric='_system')
+                        
+# Get a list of all the API key ID's names
+apiKeyIds = []
+for api in client.list_all_api_keys():
+    apiKeyIds.append(api["keyid"])
 
-# Remove api key
-remove = client.remove_api_key('id1')
+# Check wether the chosen_api_key exists or not, comparing it to the API key list
+for apiElement in apiKeyIds:
+    if apiElement == CHOSEN_API_KEY_ID:
+        API_ACTIVE = True
+        break
 
-print(remove)
+# Checks the specified API key
+if API_ACTIVE:
+    # Removes the chosen API key
+    try:
+        print("Remove API Key: ", client.remove_api_key(CHOSEN_API_KEY_ID))
```

Comment:
```suggestion
        print("Remove API key: ", client.remove_api_key(CHOSEN_API_KEY_ID))
```

Reply:
Thanks for the suggestions @diana-macrometa. Updated with suggestions.